<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры устройств</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Устройство</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Значение</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Бит</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <widget class="QComboBox" name="comboBox_9">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QComboBox" name="comboBox_12">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBox_4">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QComboBox" name="comboBox_11">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="comboBox_6">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QComboBox" name="comboBox_34">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QComboBox" name="comboBox_15">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="comboBox_8">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QComboBox" name="comboBox_14">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QComboBox" name="comboBox_13">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QComboBox" name="comboBox_16">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="comboBox_5">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QComboBox" name="comboBox_35">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="comboBox_7">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox_2">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox_3">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QComboBox" name="comboBox_10">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QComboBox" name="comboBox_36">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>141</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="5" column="0">
     <widget class="QComboBox" name="comboBox_23">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="comboBox_20">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QComboBox" name="comboBox_33">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QComboBox" name="comboBox_37">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox_19">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QComboBox" name="comboBox_26">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QComboBox" name="comboBox_24">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QComboBox" name="comboBox_28">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox_18">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QComboBox" name="comboBox_32">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QComboBox" name="comboBox_30">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QComboBox" name="comboBox_27">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="comboBox_21">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QComboBox" name="comboBox_29">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="comboBox_22">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QComboBox" name="comboBox_31">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QComboBox" name="comboBox_25">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QComboBox" name="comboBox_38">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QComboBox" name="comboBox_39">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>61</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="11" column="0">
     <widget class="QSpinBox" name="spinBox_12"/>
    </item>
    <item row="16" column="0">
     <widget class="QSpinBox" name="spinBox_17">
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSpinBox" name="spinBox_4"/>
    </item>
    <item row="0" column="0">
     <widget class="QSpinBox" name="spinBox">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QSpinBox" name="spinBox_16"/>
    </item>
    <item row="1" column="0">
     <widget class="QSpinBox" name="spinBox_2"/>
    </item>
    <item row="7" column="0">
     <widget class="QSpinBox" name="spinBox_8"/>
    </item>
    <item row="13" column="0">
     <widget class="QSpinBox" name="spinBox_14"/>
    </item>
    <item row="12" column="0">
     <widget class="QSpinBox" name="spinBox_13"/>
    </item>
    <item row="5" column="0">
     <widget class="QSpinBox" name="spinBox_6"/>
    </item>
    <item row="14" column="0">
     <widget class="QSpinBox" name="spinBox_15"/>
    </item>
    <item row="10" column="0">
     <widget class="QSpinBox" name="spinBox_11"/>
    </item>
    <item row="6" column="0">
     <widget class="QSpinBox" name="spinBox_7"/>
    </item>
    <item row="9" column="0">
     <widget class="QSpinBox" name="spinBox_10"/>
    </item>
    <item row="8" column="0">
     <widget class="QSpinBox" name="spinBox_9"/>
    </item>
    <item row="4" column="0">
     <widget class="QSpinBox" name="spinBox_5"/>
    </item>
    <item row="17" column="0">
     <widget class="QSpinBox" name="spinBox_18">
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSpinBox" name="spinBox_3"/>
    </item>
    <item row="18" column="0">
     <widget class="QSpinBox" name="spinBox_19">
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="m_pButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
